### ISSUE TITLE
Right-Alt key issue on non-english Windows systems

### ISSUE THREAD

**Issue by aaaaaaaCatOnMyKeyboard** [libsdl-org/SDL#5685]
I'm not sure if this is already known, but applications which use SDL for keyboard inputs on non-english Windows systems, cannot properly handle the Right-Alt key and recognise it as the key combination Left-Ctrl + Right-Alt.
The reason for this, is that the Ctrl-Alt key on a US keyboard sends the same scancode as the Right-Alt (AltGr) key on non-english systems. 
I've observed this for the first time in DOSBox-X and mentioned it here:
[https://github.com/joncampbell123/dosbox-x/issues/915](https://github.com/joncampbell123/dosbox-x/issues/915)

some further investigations shows, that DOSBox + forks and other applications like RetroArch for example share the same problem.
Is there a chance for a fix?

**Comment by gil938271** [comment]
yeah it is not solved. mouse works but the keyboard does not it doesn't even show an error. i have a xiaomi phone and the "latest" win10 is that an issue? it seen to be a gboard problem i had the same issue when this feature was on Linux only feature. Samsung keyboard worked gboard didn't

**Comment by slouken** [comment]
We are scoping work for the SDL 3.2.0 release, so please let us know if this is a showstopper for you.
